Project-Descriptions-and-Plans/CV32E4_TBD/Project_Concept_for_CV32E4_TBD_Nov-27_2023.md

```diff
@@ -0,0 +1,250 @@
+# OpenHW Project Concept Proposal: CV32E4_TBD
+*The PC proposal introduces the project and explains the market drivers and the "why"*
+
+# Title of Project - "CV32E4_TBD"
+# Project Concept Proposal
+## Date of proposal - 2023-11-27
+## Author(s) - Manuel PEZZIN (CEA)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is a continuation of CV32E40Pv2 project.
+Main goal is to support official RVB (and a part of RVK) and (upcoming) RVP
+extensions while keeping backward compatibility with CV32E40Pv1/v2 (eg XPULP
```

Comment:
are you planning to be LEC with v1? or SEQ? or neither of? what is the TRL you expect to achieve?

Reply:
We want to keep functional equivalence only. We could try to keep LEC with v1 for basic configuration (without XPULP, RVB, RVP) but I don't want LEC being a strong requirement.

Reply:
ok makes sense - let's aim for SEQ maybe?

Reply:
SEQ will definitely be useful during the first round of RTL modifications. This first step will mainly consist in ALU code refactoring (resource sharing, deduplication) and preparing it for addition of new features from RVB/RVP.
Maintaining SEQ after this first phase would require having a design configuration that is strictly equivalent to the most complete one from CV32E40Pv2. I don't know how to manage this properly yet but we should think about it.